### PR TITLE
`[Issue #2577]` update text style tokens

### LINKED ISSUE #2577: Update Text Style tokens
Update the design tokens

### PR DESCRIPTION
**Pull request by Da-Colon**
Closes #2577 

@decentdao/design if yall can take a look over the preview and see if anything looks off putting, in particular the text fonts. You can also view the `Commits` Tab above for the more part shows what I textStyle I replaced with what. 

Few other things that were updated here:
- Global Font was misnamed, so fixed that
- Added new `woff` and `woff2` formats for `DM Mono`
- update FontFace with new font
- found and updated very old textStyles left in
- removed all `body-base` textStyles and set to global textStyle `body-small`

### CODE REVIEW

**Comment by Da-Colon** [comment]
@decentdao/engineering This introduces new textStyle tokens into theme. All the designs are using these new style names so if you see any old ones in the design or codebase call it out for what it should be.

**Review by mudrila — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -127,11 +127,17 @@
...
       src:
         url('/fonts/DMSans-VariableFont_opsz,wght.woff2') format('woff2'),
         url('/fonts/DMSans-VariableFont_opsz,wght.woff') format('woff');
     }
+    @font-face {
+      font-family: 'DM Mono';
+      src:
+        url('/fonts/DMMono-Regular.woff2') format('woff2'),
+        url('/fonts/DMMono-Regular.woff') format('woff');
+    }
```
Comment (adamgall) [inline index.html:140]: Why did we need to add this font (and the actual font files)? Was it actually missing? I don't see where it's being used.

Comment (xanaramoss) [reply index.html:140]: This font substitutes the space mono. So we stay in the same family (DM sans).
We are using it for Snippets/helper text.

Comment (adamgall) [reply index.html:140]: Awesome thank you @xanaramoss !

**Review by adamgall — APPROVED** [review]

**Comment by nicolaus-sherrill** [comment]
Hey @Da-Colon we reviewed this morning and the only error was [this commit](https://github.com/decentdao/decent-interface/pull/2585/commits/09fa52dd2fc52abb95992d17c0a7f36c8c302860). Should be heading-xlarge

**Pull request merged**